helm/superset/templates/secret-env.yaml

```diff
@@ -30,10 +30,17 @@ metadata:
 type: Opaque
 stringData:
     REDIS_HOST: {{ tpl .Values.supersetNode.connections.redis_host . | quote }}
+    REDIS_USER: {{ .Values.supersetNode.connections.redis_user | quote }}
     {{- if .Values.supersetNode.connections.redis_password }}
     REDIS_PASSWORD: {{ .Values.supersetNode.connections.redis_password | quote }}
     {{- end }}
     REDIS_PORT: {{ .Values.supersetNode.connections.redis_port | quote }}
+    REDIS_PROTO: {{ if .Values.supersetNode.connections.redis_ssl.enabled }}"rediss"{{ else }}"redis"{{ end }}
```

Comment:
This has broken those w/o generating the default secret, which is a very common use case to manage secrets separately (rather than relying on the Helm chart).

This kind of backward compatibility should have been implemented somewhere else, and it's not a good practice not to even document/mention this feature/behavior change at all.

Reply:
The solution was to add the new `REDIS_PROTO` in the managed secret.  However, it was a hassle to root cause it, due to lacking of documentation.